src/util/Compiler.h

```diff
@@ -21,8 +21,12 @@ Copyright_License {
 }
 */
 
-#ifndef COMPILER_H
-#define COMPILER_H
+#ifndef UTIL_COMPILER_H
+#define UTIL_COMPILER_H
```

Comment:
Why???

Reply:
In my job we use cpplint (https://en.wikipedia.org/wiki/Cpplint) as sytle checker - we use an adapted Google style for the projects. In this you can find:

# The #define Guard

All header files should have #define guards to prevent multiple inclusion. The format of the symbol name should be <PROJECT>_<PATH>_<FILE>_H_.

To guarantee uniqueness, they should be based on the full path in a project's source tree. For example, the file foo/src/bar/baz.h in project foo should have the following guard:

```
#ifndef FOO_BAR_BAZ_H_
#define FOO_BAR_BAZ_H_
...
#endif  // FOO_BAR_BAZ_H_
```
We don't use the Project name as head for this to make it a little bit shorter, but with path name with underlines. So the define should be XCSOAR_UTIL_COMPILER_H - as you defined it for example in the Math folder (FastMath.hpp:  
```
#ifndef XCSOAR_MATH_FASTMATH_HPP
#define XCSOAR_MATH_FASTMATH_HPP
```
But I think, you want to do this in the same way (with XCSOAR_ as head), but you mean to do this renaming in an own commit - not byside from an other explonation, isn't it?

Reply:
Whether this is an improvement or not, and whether changing this for existing code or not is necessary - this doesn't matter, because you did not propose to change the names of include guards. You hid this change inside an unrelated commit.